pages/blog/index.page.tsx

```diff
@@ -27,23 +28,23 @@ const BlogHome = () => {
   } else if (error) {
     content = (
       <pre>
+        {/* @ts-ignore */}
         <code>{error}</code>
       </pre>
     )
   } else if (data) {
     content = (
       <>
+        {/* @ts-ignore */}
```

Comment:
Let's type and re-export `data.items` instead of littering our codebase with ignores.

Reply:
Done.